packages/blade/src/components/List/_decisions/decisions.md

```diff
@@ -0,0 +1,219 @@
+  - [Ordered List](#ordered-list)
+    - [Ordered List with 3 levels of nesting](#ordered-list-with-3-levels-of-nesting)
+    - [Ordered List with 3 levels of nesting and _filled background_ numbers](#ordered-list-with-3-levels-of-nesting-and-filled-background-numbers)
+    - [Ordered List with inline Link in items](#ordered-list-with-inline-link-in-items)
+- [API](#api)
+  - [List Component](#list-component)
+  - [List.Item Component](#listitem-component)
+  - [List.Item.Link Component](#listitemlink-component)
+  - [Sample Usage](#sample-usage)
+  - [Alternative APIs](#alternative-apis)
+    - [Hierarchial without explicit nested `List` grouping](#hierarchial-without-explicit-nested-list-grouping)
+    - [Hierarchial with a `nested` prop](#hierarchial-with-a-nested-prop)
+    - [Non-Hierarchial with a `level` prop](#non-hierarchial-with-a-level-prop)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+
+## Design
+- [Figma - List](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=23205%3A446859&t=dT849yOxgX5WHgby-4)
+
+### Unordered List
+#### Unordered List with 3 levels of nesting
+
+<img src="./unordered-list.png" width="380" />
+
+#### Unordered List with custom icons for bullets
+
+<img src="./unordered-list-icon.png" width="380" />
+
+#### Unordered List with inline Link in items
+
+<img src="./unordered-list-link.png" width="380" />
+
+### Ordered List
+#### Ordered List with 3 levels of nesting
+
+<img src="./ordered-list.png" width="380" />
+
+#### Ordered List with 3 levels of nesting and _filled background_ numbers
+
+<img src="./ordered-list-filled.png" width="380" />
+
+#### Ordered List with inline Link in items
+
+<img src="./ordered-list-link.png" width="380" />
+
+
+## API
+
+### List Component
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| children | `List.Item` | `undefined` | List items to render. | ✅ |
+| variant | `ordered`, `unordered` | `unordered` | Sets the list as ordered or unordered. |  |
+| isOrderedFilled | `boolean` | `false` | Sets whether the numbers in ordered list have a filled background.  |  |
+| icon | `BladeIcon` | `undefined` | Icon to be used for the items in the list. This can only be set when `variant` is `unordered`. |  |
+| size | `small`, `medium` | `medium` | Size of the list items. |  |
+
+### List.Item Component
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| children | `string`, `List.Item.Link`, `List` | `undefined` | String to render in the list item. You may also pass a `List.Item.Link` to render an inline link. You may also pass a `List` component which contains its own `List.Item` components to create a nested list.| ✅ |
+| icon | `BladeIcon` | `undefined` | Icon to be used for the items in the list. This can only be set when `variant` is `unordered`. |  |
+
+### List.Item.Link Component
```

Comment:
That's mainly to control the size of the Link so that it remains consistent with the rest of the font sizes of the list item (we offer 2 font sizes for list item)